### DISCUSSION TITLE
Local interface IP was not shown in the list

### DISCUSSION THREAD

**Discussion by ForsakenRei** [aceberg/WatchYourLAN#114] [Q&A] [answered]
I'm running a docker instance inside of a PVE VM (Oracle Linux 9 if that matters). `ip a show ens18` shows the local IP(let's say it's 192.168.1.24) of that VM, so in the compose file I set `IFACE=ens18`. All other devices on my network, wired or wireless, can be detected normally, but the host machine IP 192.168.1.24 never shows up.

Is this by design with ARP scan?

**Comment by SteveClement** [comment]
I can also confirm this in my setup. Perhaps it is by design or there is a certain limitation when scanning from a Docker container. Not sure.

**Comment by aceberg** [comment, marked as answer]
Yes, it is how `arp-scan` works.

**Comment by lubomben** [comment]
Is it possible to show in the list also the IP of the host where WatchYouLAN is running (in particolar for docker container with host network mode)? If yes, how?

**Reply by aceberg** [reply]
> Is it possible to show in the list also the IP of the host where WatchYouLAN is running (in particolar for docker container with host network mode)? If yes, how?

Not yet. Maybe I'll add the option later.

**Reply by jacobslusser** [reply]
I know this discussion is a bit old, but did you ever circle back to adding this? Would be nice to have.

**Reply by DrSpaldo** [reply]
> I know this discussion is a bit old, but did you ever circle back to adding this? Would be nice to have.

Agreed. It would be great for the current interface to be displayed in the list. I keep creating new services using the WatchYourLAN IP address, because it looks available... hahaha

**Reply by simonwiles** [reply]
Yeah, I would welcome this option too.  Would it be a good thing to file an FR issue for?
